apps/src/lib/wallet/mod.rs

```diff
@@ -115,6 +115,56 @@ impl Wallet {
         (alias.into(), key)
     }
 
+    /// Store the validator address in the wallet. Generate keypair
```

Comment:
should this be just
```suggestion
    /// Generate keypair
```
?